src/components/tooltip/Tooltip.tsx

```diff
@@ -217,8 +219,27 @@ class Tooltip extends React.Component<TooltipProps, State> {
             stopBubble,
             text,
             theme,
+            intl,
         } = this.props;
 
+        let ariaLabel;
+        let tooltipText;
+
+        if (text) {
+            if (typeof text === 'string') {
+                tooltipText = text;
+            } else if (React.isValidElement(text) && getProp(text, 'props')) {
```

Comment:
What is expected behavior for tooltips with not just a single element passed in? e.g. ones with a title and a body?

`GuideTooltip` is an example of this, but there are multiple variations of this in the codebase.

Reply:
Pretty God question

Reply:
This logic should work fine for `CheckboxTooltip`  and `ImageTooltip`. The `GuideTooltip` is tricky because it builds a pretty complex child component in the `text` prop. It might make sense to address this in a follow-up. Theoretically, the logic above shouldn't impact (good or bad) it.

Reply:
Where can I find a real example of those kind of Tooltips? (ex. GuideTooltip)

```
For those Tooltips I guess it should have a `aria-describedBy` relation and `aria-hidden=false`
this strategy takes care of that (but Im needing a real example to test it)
```

Reply:
Shared via Slack.